pkg/controller/volume/persistentvolume/metrics/metrics.go

```diff
@@ -105,6 +106,13 @@ var (
 			Help: "Total volume operation erros",
 		},
 		[]string{"operation_name", storageClassLabel, volumeLabel})
+
+	usedPVCsMetric = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "pvc_used_total_count",
+			Help: "Total number of PVCs in use by Pods",
+		},
+		[]string{volumeLabel})
```

Comment:
Dtto, volume name is not a good label.

Reply:
Probably no label. However, I don't like where this metric is placed. I don't think it's possible to get total count of PVCs that are used from PV controller without lot of check. @gnufied, is there a better place? Can we collect this metric in kubelet (with node name as label)? One NFS used on several nodes could be counted multiple times in that case, however it might be better metric than nr. of used PVCs - nr. of volumes of each volume plugin for each node.

Reply:
I would like to see this metric with "node" and plugin type as dimension. One catch though is - we will have to use custom collector interface for metrics that are somewhat ephemeral. 

Can we use `pvc_protection` controller as a place where this metric is calculated?

Reply:
@jsafrane, @gnufied: considering that this PR is already large, would you be OK if I postpone this metric to the next PR?